src/sentry/conf/server.py

```diff
@@ -487,7 +487,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     Queue('events.reprocess_events', routing_key='events.reprocess_events'),
     Queue('events.save_event', routing_key='events.save_event'),
     Queue('files.delete', routing_key='files.delete'),
-    Queue('incidents.notify', routing_key='incidents.notify'),
+    Queue('incidents', routing_key='incidents'),
```

Comment:
Any chance of the old queue having jobs stuck in it when the workers reboot?

Reply:
I'm not super concerned here if we lose a job, since we're still in unreleased mode.